relay-dynamic-config/src/feature.rs

```diff
@@ -1,30 +1,62 @@
+use std::borrow::Cow;
+
 use serde::{Deserialize, Serialize};
 
 /// Features exposed by project config.
-#[derive(Clone, Copy, Debug, Eq, PartialEq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash)]
 pub enum Feature {
     /// Enables ingestion and normalization of profiles.
-    #[serde(rename = "organizations:profiling")]
     Profiling,
     /// Enables ingestion of Session Replays (Replay Recordings and Replay Events).
-    #[serde(rename = "organizations:session-replay")]
     SessionReplay,
     /// Enables data scrubbing of replay recording payloads.
-    #[serde(rename = "organizations:session-replay-recording-scrubbing")]
     SessionReplayRecordingScrubbing,
     /// Enables device.class synthesis
     ///
     /// Enables device.class tag synthesis on mobile events.
-    #[serde(rename = "organizations:device-class-synthesis")]
     DeviceClassSynthesis,
     /// Unused.
     ///
     /// This used to control the initial experimental metrics extraction for sessions and has been
     /// discontinued.
-    #[serde(rename = "organizations:metrics-extraction")]
     Deprecated1,
```

Comment:
Why not remove `Deprecated1`?

Reply:
I was planning to do that in a follow-up PR, to keep the PR scope small and focused. No reason besides that.